flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -562,27 +566,30 @@ public CheckpointingMode getCheckpointingMode() {
     }
 
     /**
-     * Sets the state backend that describes how to store and checkpoint operator state. It defines
-     * both which data structures hold state during execution (for example hash tables, RockDB, or
-     * other data stores) as well as where checkpointed data will be persisted.
+     * Sets the state backend that describes how to store operator. It defines the data structures
+     * that hold state during execution (for example hash tables, RocksDB, or other data stores).
```

Comment:
I think there is a mistake in this change: shouldn't it be "how to checkpoint operator state"?

ditto: `StreamExecutionEnvironment`

Reply:
From a user perspecitve the state backend stores state locally while checkpoint storage defines how it is checkpointed. I understand under the hood its fuzzier than that but I think this javadoc is correct.